core/rint/test/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@
 # For the licensing terms see $ROOTSYS/LICENSE.
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
-ROOT_ADD_GTEST(TTabComTests TTabComTests.cxx LIBRARIES Rint)
+if(NOT minimal)
```

Comment:
Is that really the best test?  What 'feature(s)' is missing that make the test not working?